pymc/logprob/basic.py

```diff
@@ -445,8 +445,8 @@ def conditional_logp(
 
     .. math::
 
-        \log p(Y = y, \Sigma^2 = \sigma^2) =
-            \log p(Y = y \mid \Sigma^2 = \sigma^2) + \log p(\Sigma^2 = \sigma^2)
+        \log p_{Y,s^2}(y, \sigma^2) =
+            \log p_{Y \mid s^2}(y \mid \sigma^2) + \log p_{s^2}(\sigma^2)
```

Comment:
```suggestion
            \log p_{Y \mid \sigma^2}(y \mid s^2) + \log p_{\sigma^2}(s^2)
```
Idem as the comment immediately above